src/app/modules/wallet/pages/account-voluntary-exit/account-voluntary-exit.component.ts

```diff
@@ -0,0 +1,139 @@
+  styleUrls: ['./account-voluntary-exit.component.scss'],
+})
+export class AccountVoluntaryExitComponent
+  extends BaseComponent
+  implements OnInit {
+  constructor(
+    private walletService: WalletService,
+    private activatedRoute: ActivatedRoute,
+    private snackMsgService: MatSnackBar,
+    private formBuilder: FormBuilder
+  ) {
+    super();
+  }
+  exitAccountFormGroup: FormGroup | undefined | null;
+
+  keys: Account[] = [];
+
+  toggledAll = new FormControl(false);
+
+  ngOnInit(): void {
+    const publicKey = this.activatedRoute.snapshot.queryParams.publicKey;
+    this.exitAccountFormGroup = this.formBuilder.group(
+      {
+        confirmation: [
+          '',
+          [Validators.required, UtilityValidator.MustBe('agree')],
+        ],
+      },
+      {
+        validators: UtilityValidator.MustHaveMoreThanOneControl,
+      } as AbstractControlOptions
+    );
+
+    this.walletService
+      .accounts()
+      .pipe(map((x) => this.searchbyPublicKey(publicKey, x.accounts)))
+      .subscribe((accs) => {
+        accs.forEach((acc) => {
+          this.keys.push(acc);
+        });
+      });
+  }
+
+  toggleChange(selectionList: MatSelectionList, ev: MatCheckboxChange): void {
+    if (ev.checked) {
+      selectionList.selectAll();
+      this.toggledAll.setValue(true);
+      selectionList.selectedOptions.selected.forEach((x) => {
+        if (
+          this.exitAccountFormGroup &&
+          !this.exitAccountFormGroup?.get(x.value)
+        ) {
+          this.exitAccountFormGroup.addControl(
+            x.value,
+            this.formBuilder.control(x.value)
+          );
+        }
+      });
+    } else {
+      selectionList.selectedOptions.selected.forEach((x) => {
+        if (this.exitAccountFormGroup?.get(x.value)) {
+          this.exitAccountFormGroup.removeControl(x.value);
+        }
+      });
+      selectionList.deselectAll();
+      this.toggledAll.setValue(false);
+    }
+  }
+
+  selectionChange(ev: MatSelectionListChange): void {
+    if (this.exitAccountFormGroup?.get(ev.options[0].value)) {
+      this.exitAccountFormGroup.removeControl(ev.options[0].value);
+    } else {
+      this.exitAccountFormGroup?.addControl(
+        ev.options[0].value,
+        this.formBuilder.control(ev.options[0].value)
+      );
+    }
+  }
+
+  confirmation(): void | boolean {
+    if (this.exitAccountFormGroup?.invalid) {
+      return false;
+    }
+    const request = {
+      publicKeys: this.keys.map((x) => x.validatingPublicKey),
+    } as AccountVoluntaryExitRequest;
+
+    this.walletService.exitAccounts(request).subscribe((x) => {
+      const exitedKeys =
+        Object.keys(this.exitAccountFormGroup?.controls ?? {}).length - 1;
+      this.snackMsgService.open(
+        `Successfully exited ${exitedKeys} key(s)`,
+        'Success',
+        {
+          direction: 'rtl',
+          politeness: 'assertive',
+          duration: 4000,
+        }
+      );
```

Comment:
Can we make this snackbar and the below snackbar consistent with each other, as they both indicate a successful operation? Maybe define some utility service to make the code DRY?

https://github.com/prysmaticlabs/prysm-web-ui/blob/1d44a700eda83c7214cf97ecf6ef9118628ac404/src/app/modules/wallet/pages/import/import.component.ts#L49-L51